manager/types/job.go

```diff
@@ -57,6 +57,34 @@ type CreatePreheatJobRequest struct {
 	SchedulerClusterIDs []uint         `json:"scheduler_cluster_ids" binding:"omitempty"`
 }
 
+type CreateDeleteTaskJobRequest struct {
+	BIO                 string          `json:"bio" binding:"omitempty"`
+	Type                string          `json:"type" binding:"required"`
+	Args                DeleteTasksArgs `json:"args" binding:"omitempty"`
+	Result              map[string]any  `json:"result" binding:"omitempty"`
+	UserID              uint            `json:"user_id" binding:"omitempty"`
+	SchedulerClusterIDs []uint          `json:"scheduler_cluster_ids" binding:"omitempty"`
+}
+
+type DeleteTasksArgs struct {
+	TaskID string `json:"task_id" binding:"required"`
+}
+
+type CreateListTasksJobRequest struct {
+	BIO                 string         `json:"bio" binding:"omitempty"`
+	Type                string         `json:"type" binding:"required"`
+	Args                ListTasksArgs  `json:"args" binding:"omitempty"`
+	Result              map[string]any `json:"result" binding:"omitempty"`
+	UserID              uint           `json:"user_id" binding:"omitempty"`
+	SchedulerClusterIDs []uint         `json:"scheduler_cluster_ids" binding:"omitempty"`
+}
+
+type ListTasksArgs struct {
+	TaskID  string `json:"task_id" binding:"required"`
+	Page    int    `json:"page" binding:"omitempty,gte=1"`
```

Comment:
`Page` and `PerPage` are required by many requests. We can define a common request argument to reuse, but that's not the focus of this PR. We can address it in a subsequent refactoring.

Reply:
Well, there is a `page` and `perpage` parameter in the task itself, but this is mainly for handling `page` and `perpage` in the structure returned within the task result, and I'll consider defining a generic version in subsequent implementations as well.